gallery/src/styles/NxList/NxListPage.tsx

```diff
@@ -124,6 +124,32 @@ const NxListPage = () =>
               would only be a single list item which contains the error alert.
             </td>
           </tr>
+          <tr className="nx-table-row">
+            <td className="nx-cell"><code className="nx-code">.nx-list__item--clickable</code></td>
+            <td className="nx-cell"/>
+            <td className="nx-cell">Modifier of <code className="nx-code">.nx-list__item</code></td>
+            <td className="nx-cell">
+              Applied to <code className="nx-code">.nx-list__item</code> this allows for the correct styling of
+              clickable lists. Ensure that the <code className="nx-code">.nx-list</code> modifier
+              {' '}<code className="nx-code">.nx-list--clickable</code> is also used.
+            </td>
+          </tr>
+          <tr className="nx-table-row">
+            <td className="nx-cell"><code className="nx-code">.nx-list__btn</code></td>
+            <td className="nx-cell"/>
```

Comment:
Should these two have convenience components?

Reply:
> Should these two have convenience components?

I'm not certain if they _should_, but they certainly can: https://github.com/sonatype/sonatype-react-shared-components/pull/226/commits/bcbc7f97a37439bead92cdc57ec15a87f4e6ac29